src/Files.App/UserControls/InnerNavigationToolbar.xaml

```diff
@@ -105,7 +105,7 @@
 					Width="Auto"
 					MinWidth="40"
 					AccessKey="X"
-					AutomationProperties.AutomationId="InnerNavigationToolbarCutButton"
+					AutomationProperties.AutomationId="{x:Bind Commands.CutItem.AutomationName}"
```

Comment:
AutomationId should not be part of the command.

Reply:
You are right. It's done.